packages/dd-trace/src/profiling/util.js

```diff
@@ -0,0 +1,18 @@
+'use strict'
+
+function coalesce (...args) {
```

Comment:
I don't think common programming terms like "coalesce", "factory", "proxy" etc should be explained every time they are encountered.

Reply:
This operation isn't particularly common in JavaScript, and it takes at least a little bit of a mental leap for me to go from that word without any extra context (and with a terse implementation using the uncommonly-used `Array#find()`) to "nullish coalescing".

Having now made that mental leap, it now makes perfect sense, but a quick comment like `// acts similarly to null coalescing operator` would be super helpful for future readers of this code.

Reply:
We do have the `koalas` library in the tracer which I could use instead as well. Let me switch to that since looking at a library doc would make this very clear.

Reply:
SGTM. That was super helpful when I saw it previously.